### ISSUE TITLE
[ bug ] Pack fails to build custom version of the Idris compiler

### ISSUE THREAD

**Issue by stefan-hoeck** [stefan-hoeck/idris2-pack#224]
In order to speed up building the Idris compiler, pack tries to use an already installed version of Idris instead of bootstrapping the hole thing. This works reasonably well with `pack switch latest`, but fails, if people edit their `pack.toml` files manually unless they specifically use `--bootstrap` as a command line option. See also the discussion in #220 .

I'm sure we can be smarter than that.

**Comment by buzden** [comment]
That's strange. I set alternative Idris version in local `pack.toml` files many times, never passed the `--bootstrap` option and still never ran into this problem. How can this be?

**Comment by stefan-hoeck** [comment]
> That's strange. I set alternative Idris version in local `pack.toml` files many times, never passed the `--bootstrap` option and still never ran into this problem. How can this be?

Do you have `bootstrap` set to true in your `pack.toml` file?

**Comment by buzden** [comment]
> > That's strange. I set alternative Idris version in local `pack.toml` files many times, never passed the `--bootstrap` option and still never ran into this problem. How can this be?
> 
> Do you have `bootstrap` set to true in one of your `pack.toml` files?

No, it is set to `false` in `~/.pack/user/pack.toml` and it not reset in local ones.
